cfme/tests/control/test_basic.py

```diff
@@ -44,17 +45,181 @@
     # Needs working input/select mutability
 ]
 
-CONTAINER_POLICIES = [
+POLICIES = [
+    explorer.HostCompliancePolicy,
+    explorer.VMCompliancePolicy,
     explorer.ReplicatorCompliancePolicy,
     explorer.PodCompliancePolicy,
     explorer.ContainerNodeCompliancePolicy,
     explorer.ContainerImageCompliancePolicy,
+    explorer.HostControlPolicy,
+    explorer.VMControlPolicy,
     explorer.ReplicatorControlPolicy,
     explorer.PodControlPolicy,
     explorer.ContainerNodeControlPolicy,
     explorer.ContainerImageControlPolicy
 ]
 
+CONTROL_POLICIES = [
+    explorer.HostControlPolicy,
+    explorer.VMControlPolicy,
+    explorer.ReplicatorControlPolicy,
+    explorer.PodControlPolicy,
+    explorer.ContainerNodeControlPolicy,
+    explorer.ContainerImageControlPolicy
+]
```

Comment:
I made separate list for control policies because for test_assign_two_random_events_to_control_policy needs only control policies